docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -276,13 +317,192 @@ Relative times can be in any of the following formats (again, note that you can
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+#### Undo column deletion
+
+1. Using the same keyspace as the previous scenarios, alter your table by dropping a column.
+
+    ```sql
+    alter table pitr.employees drop salary;
+    select * from pitr.employees;
+    ```
+
+    ```output
+     employee_no | name           | department
+    -------------+----------------+-----------
+            1223 |   Lucille Ball | Operations
+            1224 | John Zimmerman |      Sales
+            1221 |     John Smith |  Marketing
+            1222 |    Bette Davis |      Sales
+
+    (4 rows)
+    ```
+
+1. Now restore back to a time before this table was altered, as in [Restore from a relative time](#restore-from-a-relative-time).
+
+1. Due to a ycqlsh caching issue, to see the effect of this change, you will need to drop out of your current ycqlsh session and log back in.
+
+1. Check to see that the salary column is back.
+
+    ```sql
+    ycqlsh:pitr> select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+### Undo index creation
+
+1. Create an index on the table from the previous examples.
+
+    ```sql
+    use pitr;
+    create index t1_index on employees (employee_no);
+    describe index t1_index;
+    ```
+
+    ```output
+    CREATE INDEX t1_index ON pitr.employees (employee_no)
+    WITH transactions = {'enabled': 'true'};
+    ```
+
+1. Now restore back to a time before this index was created.
+
+    **Note**: Due to a ycqlsh caching issue, to see the effect of this change, you will need to drop out of your current ycqlsh session and log back in.
+
+    ```sh
+    ./bin/ycqlsh -e 'use pitr; describe table pitr.t1_index;'
+    ```
+
+    ```output
+    <stdin>:1:Column family u't1_index' not found
+    ```
+
+### Undo table and index deletion
+
+1. Create a second table called `dont_deleteme`.
+
+    ```sql
+    create table dont_deleteme (
+      oops integer PRIMARY KEY,
+      mistake text
+    ) with transactions = { 'enabled' : true };
+    ```
+
+1. Create an index on the table from the previous examples.
+
+    ```sql
+    use pitr;
+    create index t1_index on employees (employee_no);
+    describe index t1_index;
+    ```
+
+    ```output
+    CREATE INDEX t1_index ON pitr.employees (employee_no)
+    WITH transactions = {'enabled': 'true'};
+    ```
+
+1. Get a timestamp using one of the methods in [Restore from an absolute time](#restore-from-an-absolute-time).
```

Comment:
In the other spots we say
> Now restore back to a time before this table was created, as in Restore from a relative time.

but in here we use the absolute time phrasing
> Get a timestamp using one of the methods in Restore from an absolute time.

was that by choice? if not, can we make it consistent?

Reply:
I did it this way because I wrote this one from scratch, and wasn't really paying attention to the others. TBH the "get a timestamp, do the damage, restore to timestamp" flow is more foolproof, but adds an annoying step. Suggestions welcome here. :)